README.md

```diff
@@ -1 +1,49 @@
-# ERSU-Bot
\ No newline at end of file
+# ERSU-Bot
+A Telegram Bot for UNICT students aided at helping students find informations about ERSU services.
+
+## How does it work?
+
+##Requirements
+-Pip3
+-Python3
+
+## docker-compose
+```yaml
+version: '2'
+services:
+  ersu-bot:
+    container_name:ersubot
+    image: ghcr.io/unict-dmi/ersubot:master
```

Comment:
this should be: `ghcr.io/unict-dmi/ersu-bot `

remember that the url is `https://github.com/UNICT-DMI/ERSU-Bot/` so it is unict-dmi/ersu-bot not ersubot